homeassistant/components/switch/broadlink.py

```diff
@@ -133,6 +131,49 @@ def _send_packet(call):
                         if retry == DEFAULT_RETRY-1:
                             _LOGGER.error("Failed to send packet to device")
 
+    def _decode_packet(packet):
+        packet = packet.strip()
+        if packet.startswith('0000'):
```

Comment:
Actually not in this case, because legitimate Broadlink data packets have to start with `0x26` = IR, `0xb2` for RF 433Mhz, or `0xd7` for RF 315Mhz. Source: https://github.com/mjg59/python-broadlink/blob/master/protocol.md

I deliberately added no special marker because it was not ambiguous.